plugins/chat/config/routes.rb

```diff
@@ -41,6 +41,10 @@
     get "/channels/:channel_id/summarize" => "summaries#get_summary"
   end
 
+  namespace :admin, defaults: { format: :json, constraints: StaffConstraint.new } do
+    post "export/messages" => "export#export_messages"
```

Comment:
I would argue that `export_messages#create` would be better, but up to you

Reply:
I'd prefer to leave it like this, this way in the future we'll be able to add other exports to this route / controller.